src/components/table/table.js

```diff
@@ -1207,9 +1212,15 @@ export default {
           },
           on: {
             click: evt => {
+              if (!sortable) {
+                return
+              }
               this.headClicked(evt, field)
```

Comment:
Head Clicked is used for more than just sorting...

Which is why the tests are failing.

You may need to pass `sortable` as the third argument to the `this.headClicked(...)` call, and then put some logic around the sorting routines in the handler (using the `sortable` argument passed)